### PR TITLE
Added togglability to Epop-message for compatibility with other addons

### PR DESCRIPTION
**Pull request by BeeJay28**
tl;dr: added new convar to toggle epop message, because other addons might have more important information delayed or dismissed because of the ceasefire message.

I added another convar in the same fashion as your own convars to be able to toggle the Epop-message at the top of the screen when the round starts. The changes are simple enough and don't really need explanation, I believe. If I am mistaken, feel free to ask. (Btw, thank you for knowing how that convar stuff works because I don't understand poo in that convar_util file of yours and I am glad I don't have to.)
Main reason for adding it is compatibility with other TTT(2) addons like the [Minigames-addon](https://steamcommunity.com/sharedfiles/filedetails/?id=2266863830) or roles like the [thief](https://steamcommunity.com/sharedfiles/filedetails/?id=2652144870).
If a server uses other addons which also utilise Epop, the information provided by those other addons often gets delayed or sometimes even completely dismissed if addons like this one use it first. Furthermore, if a static playerbase gets used to the ceasefire, the sidebar display is good enough and should have a way to open the epop-channel for compatibility.

### CODE REVIEW

**Comment by EntranceJew** [comment]
I came here to put in a similar pull request, looks like you got to it first, although I'm a little disappointed to see it hasn't been merged.

**Comment by BeeJay28** [comment]
> I came here to put in a similar pull request, looks like you got to it first, although I'm a little disappointed to see it hasn't been merged.

Bruh same xD

**Comment by ZenBre4ker** [comment]
Oh F, Im sorry. Wasnt active the last year.
Gonna take a look at it and will probably merge it.

**Review by ZenBre4ker — APPROVED** [review]

**Comment by ZenBre4ker** [comment]
Thanks for your contribution. Uploaded it to Steam. Better late than never I guess. xD
